tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -274,6 +279,7 @@ def test_should_response_200_for_null_start_date(self):
             "fileloc": __file__,
             "file_token": FILE_TOKEN,
             "is_paused": None,
+            "is_active": None,
```

Comment:
is_active should never be None .

Reply:
I think that in this case the DAG object hasn't been sync'd to the DB, which is why it returns None (the same happens with `is_paused` here). 

From my understanding it should never be None in the real world, but in the current testing set up this makes sense?